### PR TITLE
feat(ui): Implement Unified view graph

### PR DESCRIPTION
**Pull request by bdauria**
# Pull Request Template

## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [x] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by bdauria** [comment]
This feature will be covered by image snapshot testing afterwards.

File: webpack.config.js [hunk 1]
```diff
@@ -23,10 +24,8 @@ module.exports = merge(baseConfig, extractCssConfig, {
       template: './www/front_src/public/index.html',
       filename: '../index.html',
     }),
+    new BundleAnalyzerPlugin(),
```
Comment (Thebarda) [inline webpack.config.js]: Do not forget to remove this :)

Comment (bdauria) [reply webpack.config.js]: Actually I'm going to set it up in a "prod" configuration

File: webpack.config.js [hunk 2]
```diff
@@ -24,9 +24,6 @@ module.exports = merge(baseConfig, extractCssConfig, {
       filename: '../index.html',
     }),
   ],
-  resolve: {
```
Comment (jdelpierre) [inline webpack.config.js]: You remove it because now, we don't use the custom SVG files ?

Comment (bdauria) [reply webpack.config.js]: We do, but it's not a typical source module, so it's not needed here.

**Comment by bdauria** [comment]
@kduret the extracted recharts chunk size is 250kb, which is less than @material-ui (~600). I would suggest we leave it for now and decide the split strategy later.

**Pull request merged**